EndlessClient/Dialogs/ActiveDialogRepository.cs

```diff
@@ -144,6 +146,8 @@ public class ActiveDialogRepository : IActiveDialogRepository, IActiveDialogProv
 
         public Option<BarberDialog> BarberDialog { get; set; }
         
+        public Option<GuildDialog> GuildDialog { get; set; }
+
         public Option<ScrollingListDialog> HelpDialog { get; set;  }
```

Comment:
You need to also add the `.Map` to the `ActiveDialogs` collection and the `Dispose` resetting it below